tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -29,3 +29,24 @@
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.GetAzureRMExpressRouteCrossConnectionRouteTableSummaryCommand","Get-AzureRMExpressRouteCrossConnectionRouteTableSummary","0","1000","The cmdlet 'Get-AzureRMExpressRouteCrossConnectionRouteTableSummary' has been removed and no alias was found for the original cmdlet name.","Add the cmdlet 'Get-AzureRMExpressRouteCrossConnectionRouteTableSummary' back to the module, or add an alias to the original cmdlet name."
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureRMExpressRouteCrossConnectionPeeringCommand","Remove-AzureRMExpressRouteCrossConnectionPeering","0","1000","The cmdlet 'Remove-AzureRMExpressRouteCrossConnectionPeering' has been removed and no alias was found for the original cmdlet name.","Add the cmdlet 'Remove-AzureRMExpressRouteCrossConnectionPeering' back to the module, or add an alias to the original cmdlet name."
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.Automation.GetAzureRMNetworkWatcherReachabilityReport","Get-AzureRMNetworkWatcherReachabilityReport","0","1000","The cmdlet 'Get-AzureRMNetworkWatcherReachabilityReport' has been removed and no alias was found for the original cmdlet name.","Add the cmdlet 'Get-AzureRMNetworkWatcherReachabilityReport' back to the module, or add an alias to the original cmdlet name."
+"Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.AddAzureApplicationGatewayAuthenticationCertificateCommand","Add-AzureRmApplicationGatewayAuthenticationCertificate","0","3000","The type of property 'AddressPrefix' of type 'Microsoft.Azure.Commands.Network.Models.PSSubnet' has changed from 'System.String' to 'System.Collections.Generic.List`1[System.String]'.","Change the type of property 'AddressPrefix' back to 'System.String'."
```

Comment:
Are all of these cmdlets affected by this change?

Reply:
That's correct. Because all those cmdlets refer to PSSubnet class.